tests/test_clusterv2.py

```diff
@@ -384,4 +384,54 @@ def test_minimal(self):
         Connection._api_call = Mock(return_value={})
         qds.main()
         Connection._api_call.assert_called_with('POST', 'clusters/1234/clone', {'cluster_info':
-                                                                                    {'label': ['test_label1', 'test_label2']}})
\ No newline at end of file
+                                                                                    {'label': ['test_label1', 'test_label2']}})
+
+class TestClusterList(QdsCliTestCase):
```

Comment:
Seems to be fixed now, please check and let me know if you see any more discrepencies